plusplus.js

```diff
@@ -0,0 +1,24 @@
+const { createClient } = require('redis');
+
+const client = createClient();
+
+// Redis schema, key => value
+// username (str) => score (number as str)
+const initPlusPlus = async () => {
+
+   await client.connect();
+
+}
+
+// !++ @username
+const plusPlus = async (username) => {
+   const currentScoreStr = await client.get(username);
+   const newScore = (currentScoreStr) ? Number.parseInt(currentScoreStr) + 1 : 1;
+   await client.set(username, newScore);
```

Comment:
Actually, ignore the code above. Redis has an [increment operator](//redis.io/commands/incr/). It will initialize the key to `0` is it doesn't exist.

So, it should be:

```suggestion
   return await client.incr(username);
```